t/script/release/skip_broken_files.t

```diff
@@ -0,0 +1,42 @@
+use strict;
+use warnings;
+
+use Archive::Tar;
+use HTTP::Request;
+use LWP::UserAgent;
+use MetaCPAN::Script::Release;
+use Test::More;
+
+use Test::RequiresInternet ( 'metacpan.org' => 443 );
+
+my $ua = LWP::UserAgent->new(
+    parse_head => 0,
+    env_proxy  => 1,
+    agent      => 'metacpan',
+    timeout    => 30,
+);
+
+my $URL
+    = 'https://cpan.metacpan.org/authors/id/S/SH/SHULL/karma-0.7.0.tar.gz';
+my $tarball = 'tarball.tar.gz';
+
+my $req = HTTP::Request->new( GET => $URL );
+$req->header( Accept => '*/*' );
+my $res = $ua->request( $req, $tarball );
+
+my $arch = Archive::Tar->new;
+$arch->read($tarball);
+
+my $next = Archive::Tar->iter($tarball);
+while ( my $file = $next->() ) {
+    if ( $file->is_fifo ) {
+        my $skip = MetaCPAN::Script::Release->_is_broken_file($file);
```

Comment:
You should be creating a Release object and then testing the method rather than calling this as a class method.
```
my $release = MetaCPAN::Script::Release->new( ... );
ok( $release->_is_broken_file( $file ), 'The file is a broken pipe');
```